.github/workflows/setup-java-maven.yml

```
@@ -0,0 +1,32 @@
+name: Setup Java and Maven
```

actionlint (1.7.4)

"on" section is missing in workflow

(syntax-check)